Loop/Managers/TrustedTimeChecker.swift

```diff
@@ -74,15 +74,20 @@ class TrustedTimeChecker {
                 let deviceNow = Date()
                 let ntpNow = referenceTime.now()
                 let timeDelta = ntpNow.timeIntervalSince(deviceNow)
-                let timeSinceLastAlert = abs(ntpNow.timeIntervalSince(UserDefaults.standard.lastSignificantTimeChangeAlert ?? Date.distantPast))
 
-                if abs(timeDelta) > self.acceptableTimeDelta, timeSinceLastAlert > self.minimumAlertFrequency {
+                if abs(timeDelta) > self.acceptableTimeDelta {
                     self.log.info("applicationSignificantTimeChange: ntpNow = %@, deviceNow = %@", ntpNow.debugDescription, deviceNow.debugDescription)
```

Comment:
Nit: `.default` or `.error` might be more appropriate for this log. Seems pretty important.

Reply:
will use `.default`